src/jit/alloc.cpp

```diff
@@ -81,11 +83,8 @@ size_t ArenaAllocator::getDefaultPageSize()
 // ArenaAllocator::ArenaAllocator:
 //    Default-constructs an arena allocator.
 ArenaAllocator::ArenaAllocator()
-    : m_memoryManager(nullptr)
-    , m_firstPage(nullptr)
-    , m_lastPage(nullptr)
-    , m_nextFreeByte(nullptr)
-    , m_lastFreeByte(nullptr)
+    : m_memoryManager(nullptr), m_firstPage(nullptr), m_lastPage(nullptr), m_nextFreeByte(nullptr),
```

Comment:
I can change ConstructorInitializerAllOnOneLineOrOnePerLine to true (right now it's false). I think in many cases, it will still do this all on the same time thing, but in some of them it may fix it.